requirements-base.txt

```diff
@@ -70,7 +70,7 @@ urllib3[brotli]>=1.26.9
 brotli>=1.0.9
 # See if we can remove LDFLAGS from lib.sh
 # https://github.com/getsentry/sentry/pull/30094
-pyuwsgi==2.0.20.0
```

Comment:
this isn't really a typo -- 2.0.20.0 and 2.0.20 are the same version.  the actual bug is in poetry not handling version normalization correctly.  we don't support installing this repository through poetry (or even pip for that matter)

Reply:
Fair, typo is the wrong word. 🙂 
Is this something we want to keep or is it okay to drop the trailing `.0`?

I can also make a note on the issue that this appears to be a bug in Poetry.

Reply:
I just think we shouldn't bother -- we'd be "fixing" something that's not supported and is a bug in poetry